lib/bn.js

```diff
@@ -731,6 +731,45 @@ BN.prototype.uxor = function uxor(num) {
 };
 
 
+// Not ``this`` with ``width`` bitwidth
+BN.prototype.inotn = function inotn(width) {
+  assert(typeof width === 'number' && width >= 0);
+
+  var bytesNeeded = width / 26;
```

Comment:
The issue is that you comparing integers with a floating point number in a loop below, which is slower than comparing integer to integer (obviously).